Update 22.vaccines.md (#911)
greenelab/covid19-review: content/22.vaccines.md

These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.
Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].

> The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms.

Reviewer: Consider adding citation here too or rearranging the paragraph break?

Author: ```suggestion
The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
```

> The phase 1/2 trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].

Reviewer 2: ```suggestion
The phase I/II trial was a randomized, placebo-controlled study with 131 healthy adult participants in 5 treatment arms [@doi:10.1056/NEJMoa2026920].
```

Reviewer 2: Mainly to be consistent with the other text

> Participants that received the recombinant SARS-CoV-2 vaccine with or without Matrix-M adjuvant got two injections, 21 days apart.

Reviewer 3: ```suggestion
Participants that received the recombinant SARS-CoV-2 vaccine with or without the Matrix-M adjuvant got two injections, 21 days apart.
```

Primary outcomes that were assessed include reactogenicity, lab-values (serum chemistry and hematology), and anti-spike IgG levels.
Secondary outcomes measured included virus neutralization, T-cell responses, and unsolicited adverse events.
The authors reported that no serious treatment-related adverse events occurred in any of the treatment arms.
Reactogenicity was mostly absent and of short duration.
The two-dose vaccine regimen induced anti-spike IgG levels and neutralizing antibody-titers exceeding those in the convalescent plasma of symptomatic patients.
In line with the preclinical studies, the use of Matrix-M adjuvant further increased anti-spike immunoglobulin levels and induced a Th1 response.
The outcomes of this trial suggest that Novavax-CoV2373 has an acceptable safety profile and is able to induce a strong immune response with high neutralizing antibody titers.

> The phase 2 component of this phase 1/2 trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].

Reviewer 2: ```suggestion
The phase II component of this phase I/II trial was recently uploaded to an open-access repository [@doi:10.1101/2021.02.26.21252482].
```

This part of the trial was designed to identify which dosing regimen should move forward into late phase clinical trials.

> Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5ug or 25ug Novavax-CoV2373 or placebo in two doses, 21 days apart.

Reviewer: ```suggestion
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.
```

> Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.

Reviewer 2: Is the placebo just the adjuvant I assume...just glancing through the paper I think thats the case, but just wanted to confirm with you

Reviewer: Hmmm I'm not sure they explain what it was! I might leave a comment to ask the authors. There was another clinical trial of the adjuvant alone but it's not clear what they did here.

For now, I'm making a note here so we can merge this in.
```suggestion
Both younger (18-59 years) and older patients (60-84 years) were randomly assigned to receive either 5 &mu;g or 25 &mu;g Novavax-CoV2373 or placebo in two doses, 21 days apart.<!--To Do: What is placebo?-->
```

Author: I think adjuvant only was done previously already. The company uses this adjuvant for other vaccines as well so they did not have to test the adjuvant on its own anymore.

> In line with the phase 1 data, reactogenicity remained mild to moderate and of short duration. 

Reviewer 2: ```suggestion
In line with the phase I data, reactogenicity remained mild to moderate and of short duration. 
```

> Both doses were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.

Reviewer: ```suggestion
Both dose levels were able to induce high anti-spike IgG titers as well as neutralizing antibody responses after the second dose.
```

> Based on both safety and efficacy data, the 5 ug dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.

Reviewer: ```suggestion
Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.
```

> Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase 3 trial.

Reviewer 2: ```suggestion
Based on both safety and efficacy data, the 5 &mu;g dosing regimen was selected as the optimal dose regiment for the ongoing phase III trial.
```

> Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.

Reviewer 4: Is there a press release or website we can cite for this?

Reviewer 3: Same for the phase 1 data :)

Author: Yes: https://ir.novavax.com/news-releases/news-release-details/novavax-covid-19-vaccine-demonstrates-893-efficacy-uk-phase-3

> Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.

Reviewer 2: ```suggestion
Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.
```

> Although the phase 3 trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase 3 trial in the UK and South Africa.

Reviewer 2: ```suggestion
Although the phase III trial data has not been published yet, Novavax announced an efficacy of 89.3% based on their phase III trial in the UK and South Africa (SA).
```

> This trial included over 15.000 participants in the UK and 4000 participants in SA with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.

Reviewer 4: ```suggestion
This trial included over 15,000 participants in the UK and 4,000 participants in South Africa with occurrence of a PCR-confirmed symptomatic case as the primary endpoint.
```

In the first interim analysis (UK), 56 cases of COVID-19 were observed in the placebo group compared to 6 cases in the treatment group.

> Importantly, the vaccine candidate also shows significant clinical efficacy against the prevelant UK and South African variant.

Reviewer: ```suggestion
Importantly, the vaccine candidate also shows significant clinical efficacy against the prevelant UK and South African variants.
```

> Importantly, the vaccine candidate also shows significant clinical efficacy against the prevelant UK and South African variants.

Reviewer: ```suggestion
Importantly, the vaccine candidate also shows significant clinical efficacy against the prevalent UK and South African variants.
```

> The company also has initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.

Reviewer: ```suggestion
The company has also initiated the development of new constructs to select candidates that can be used as a booster against new strains and plans to initiate clinical trials for these new constructs in the second quarter of 2021.
```
